src/strategies/DeltaNeutralLp.sol

```diff
@@ -17,65 +17,81 @@ import {AccessStrategy} from "./AccessStrategy.sol";
 import {IMasterChef} from "src/interfaces/sushiswap/IMasterChef.sol";
 import {SlippageUtils} from "src/libs/SlippageUtils.sol";
 
+struct LenderInfo {
```

Comment:
This can be combined with `LendingParams` below.

Reply:
Lending params set by strategist, but LenderInfo is static info, Shouldn't be good idea to merge them.

Reply:
The LenderInfo is actually different depending on the asset/borrow combination of the strategy. You can combine them to reduce clutter and increase readability. You only need one struct to give all of the info needed to successfully borrow from aave.